ci/build_wheel.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name=$1
+package_dir=$2
+
+# Use gha-tools rapids-pip-wheel-version to generate wheel version then
+# update the necessary files
+RAPIDS_EPOCH_TIMESTAMP=$(date +%s)
```

Comment:
Hmm maybe we want this to be a bit more sophisticated. The question is, do we care if the different matrix builds for a given package produce slightly different versions? That's what will happen if we move the timestamp generation into the script. The purist in me says that's bad, although in practice it's probably not a huge deal either way.

Reply:
I can add this step back in the workflow, and then export this variable to the environment. Is that okay?

Reply:
Yes that's fine with me.